tests/InputFiles/Xcts/HeadOnBns.yaml

```diff
@@ -0,0 +1,195 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: SolveXcts
+# Check: parse
+# Timeout: 30
+
+ResourceInfo:
+  AvoidGlobalProc0: false
+  Singletons: Auto
+
+Background: &background
+  Binary:
+    XCoords: [&x_left -20., &x_right 20.]
+    ObjectA: &star_right
+      TovStar:
+        # Values taken from Sec. V.C and Fig. 8 in
+        # https://arxiv.org/abs/1708.07358.
```

Comment:
I think we should use the numbers from https://arxiv.org/pdf/2010.14518.pdf instead. It's more recent and also evolves the binaries, and allows for a simple thermal component.

Reply:
You can replace this with an EOS of your choice when generating ID :) happy to put another one here as the example. The paper you link doesn't
list the central density though, or maybe I've missed it. Also, is the thermal component implemented?

Reply:
Okay, let's just leave it as-is. Like you said, we can adjust later :)